BetterModSupport/Scripts/ModManager.cs

```diff
@@ -96,7 +96,15 @@ private void Start()
             files.ForEach(i =>
             {
                 // Iterate over each json file and store its data using the ModInfo class.
-                _installedModList.Add(new ModInfo(i));
+                _installedModList.Add(
+                    new ModInfo(FormatPathAccordingly(i)
```

Comment:
The second parenthesis ending could be rearranged here, looks odd to read it that way